src/markdown.js

```diff
@@ -67,17 +102,24 @@ export default function (config, options = {}) {
 				return defaultRender(tokens, idx, options, env, self).replace(...epubRules[rule]);
 			}
 		});
-		// md.renderer.rules.footnote_ref = function (tokens, idx, options, env, slf) {
-		// 	const id = slf.rules.footnote_anchor_name(tokens, idx, options, env, slf);
-		// 	const caption = slf.rules.footnote_caption(tokens, idx, options, env, slf);
-		// 	let refid = id;
 
-		// 	if (tokens[idx].meta.subId > 0) {
-		// 		refid += `:${tokens[idx].meta.subId}`
-		// 	}
+		md.renderer.rules.footnote_ref = function (tokens, idx, options, env, slf) {
+			let id = slf.rules.footnote_anchor_name(tokens, idx, options, env, slf);
+			let caption = slf.rules.footnote_caption(tokens, idx, options, env, slf);
+
+			let refId = id;
+			if (tokens[idx].meta.subId > 0) {
+				refId += `:${tokens[idx].meta.subId}`;
+			}
+
+			let footnotes = allFootnotes.get(env.id);
+			let footnoteLabel = tokens[idx].meta.label;
+			let footnoteText = footnotes[footnoteLabel];
+			footnoteText = md.renderInline(footnoteText);
+			footnoteText = `<span class="footnote-text">${footnoteText}</span>`
```

Comment:
```suggestion
			footnoteText = `<span class="footnote-text inline" hidden>${footnoteText}</span>`
```

(We’ll make it visible with CSS when needed)